src/types/Types.ts

```diff
@@ -0,0 +1,29 @@
+
+export interface SearchRequest {
+  keyword: string;
+}
+
+export interface User {
+  userId: string;
+  firstName: string;
+  lastName: string;
+  email: string;
+  phoneNumber: string;
+  address: string;
+}
+
+export interface DataInputRequest {
+  user: User;
+}
+
+export interface DataInputResponse {}
+
+export interface DataDisplayResponse {
```

Comment:
This is the sample payload for `DataDisplayResponse`:
```
{"userDetailsList":[{"userId":"vap1","firstName":"vaibhav","lastName":"vasant","email":"v@gmail.com","phoneNumber":"7633824597","address":"None"},{"userId":"vap2","firstName":"vaibhav","lastName":"vasant","email":"v@gmail.com","phoneNumber":"7633824597","address":"None"}]}
```
Based on that change `users` to `userDetailsList`